packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
             created
           example: 1587667174725
 
+    vod-asset:
+      type: object
+      table: vod_asset
+      required:
+        - name
+      properties:
+        id:
+          type: string
+          readOnly: true
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+          index: true
+        userId:
+          type: string
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+          index: true
+          description: owner of the asset
+        deleted:
+          type: boolean
+          description: Set to true when the asset is deleted
+        objectStoreId:
+          type: string
+          description: Object store ID where the asset is stored
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        name:
+          type: string
+          description:
+            Name of the asset. This is not necessarily the filename, can be a
+            custom name or title
+          example: filename.mp4
+        duration:
+          type: number
+          description: Duration of the asset in seconds (float)
+          example: 23.8328
+        aspectRatio:
+          type: string
+          description: Aspect ratio of the asset
+          example: 16:9
+        meta:
```

Comment:
I think this needs a `MAX_META_SIZE` in the post validation

Reply:
agreed! might need another value to check max size of key-value as well maybe? for eg. total meta size = 1MB, fields inside can be at most 100KB?

Reply:
I think there's a JSON schema way to enforce max number of elements in an array. Maybe we can enforce on the keys somehow as well.